src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -110,9 +110,9 @@ public ShooterValues interpolate(ShooterValues other, double ratio) {
 
             // Pass shots from Center zone
             // close to Stage
-            Map.entry(6.0, new ShooterValues(2000.0, 2000.0, Math.toRadians(50))),
+            Map.entry(6.0, new ShooterValues(2250.0, 2250.0, Math.toRadians(50))),
```

Comment:
We should revisit / retest the pass shot, so we're comfortable with the pass.
And also confirm that a true speaker shot from faraway is not affected by changes for the pass shot.

Reply:
Yes. I will add some into a new branch.